src/main/java/com/minecolonies/coremod/network/NetworkChannel.java

```diff
@@ -93,6 +93,9 @@ public void registerCommonMessages()
         registerMessage(++idx, LumberjackReplantSaplingToggleMessage.class, LumberjackReplantSaplingToggleMessage::new);
         registerMessage(++idx, LumberjackRestrictionToggleMessage.class, LumberjackRestrictionToggleMessage::new);
         registerMessage(++idx, LumberjackScepterMessage.class, LumberjackScepterMessage::new);
+        registerMessage(++idx, CreateColonyMessage.class, CreateColonyMessage::new);
+        registerMessage(++idx, ColonyDeleteOwnMessage.class, ColonyDeleteOwnMessage::new);
+        registerMessage(++idx, ColonyViewRemoveMessage.class, ColonyViewRemoveMessage::new);
```

Comment:
This may break universe when previous version connects to this version. Always add new messages to the end of the list, or increase network proto version

Reply:
previous version never are allowed to connect to a newer version afaik

Reply:
We add this always next to matching messages and it never was a problem.

Reply:
weird, i was able to connect against different version in non dev env while testing some small mods, might be their bug